extensions/amp-fit-text/0.1/test/test-amp-fit-text.js

```diff
@@ -53,14 +53,13 @@ describes.realWin(
         .then(() => ft);
     }
 
-    it('renders', () => {
+    it('renders', async () => {
       const text = 'Lorem ipsum';
-      return getFitText(text).then((ft) => {
-        const content = ft.querySelector('.i-amphtml-fit-text-content');
-        expect(content).to.not.equal(null);
-        expect(content.textContent).to.equal(text);
-        expect(ft.textContent).to.equal(text);
-      });
+      const ft = await getFitText(text);
+      const content = ft.querySelector('.i-amphtml-fit-text-content');
+      expect(content).to.not.be.null;
+      expect(ft.textContent).to.equal(text);
+      await ft.implementation_.unlayoutCallback();
```

Comment:
FMI: Why is this line added to test that the component renders?

Reply:
I initially added this because without this call the ResizeObserver unlistener, was not being correctly triggered, however after debugging this appears to have been related to a different issue so I have removed this line in this test.